host/src/main/java/org/area515/resinprinter/server/Main.java

```diff
@@ -59,6 +59,9 @@ public static void main(String[] args) throws Exception {
 		logger.info("=================================================================");
 
 		int port = HostProperties.Instance().getPrinterHostPort();
+		if (System.getProperty("overrideHostPort") != null) {

```

Comment:
I'm thinking we should put your override inside the HostProperties.Instance().getPrinterHostPort() or inside the HostProperties constructor itself. That way anyone else calling the method will get the true overridden host port value.